test/src/utils/kalman_multivariate.test.cpp

```diff
@@ -67,7 +67,7 @@ struct KalmanFunctionality : public ::testing::Test {
  */
 TEST_F(KalmanFunctionality, handlesZeroStateEstimate)
 {
-  ASSERT_EQ(kalman.getStateEstimate(), x0);
+  ASSERT_EQ(kalman.getStateEstimate(), x0) << "Should handle zero vector as state estimate";
```

Comment:
Maybe it would be a good Idea to encapsulate the error messages in variables. I think it is cleaner that way

Reply:
Yeah I agree. We need to have a discussion about this when Sem 2 starts to properly define the style. Perhaps creating a simple variable in the Test Struct storing the message